frontend/public/components/factory/list.jsx

```diff
@@ -200,9 +202,62 @@ export const WorkloadListHeader = props => <ListHeader>
   <ColHead {...props} className="col-lg-3 hidden-md hidden-sm hidden-xs" sortField="spec.selector">Pod Selector</ColHead>
 </ListHeader>;
 
-const Rows = ({data, expand, Row, kindObj}) => <div className="co-m-table-grid__body">
-  {data.map(obj => <Row key={obj.rowKey || obj.metadata.uid} obj={obj} expand={expand} kindObj={kindObj} />)}
-</div>;
+export class Rows extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.measurementCache = new CellMeasurerCache({
```

Comment:
`CellMeasurerCache` caches by default by row and column but it should probably be using [keyMapper](https://github.com/bvaughn/react-virtualized/blob/master/docs/CellMeasurer.md#prop-types-1) and metadata.uid